File: Uri-online-judge/1015/1015.cpp
// 1015 - Distance Between Two Points
#include <stdio.h>
#include <math.h>
int main()
{
    float x1, y1, x2, y2;
    double result;
    scanf("%f %f", &x1, &y1);
    scanf("%f %f", &x2, &y2);

    result = sqrt(pow((x2 - x1), 2.0) + pow((y2 - y1), 2.0));
    printf("%.4f\n", result);
    return 0;
}

File: Uri-online-judge/1015/1015.go
package main

import (
	"fmt"
	"math"
)

func main(){
	var x1, x2, y1, y2, val1, val2 float64
	fmt.Scanf("%f %f", &x1, &y1)
	fmt.Scanf("%f %f", &x2, &y2)
	val1 = math.Pow(x2-x1, 2)
	val2 = math.Pow(y2-y1,2)
	result := math.Sqrt(val1+val2)
	fmt.Printf("%.4f\n", result)
}
